src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -40,6 +40,9 @@ public String getUrlForWiki(String wikiName) {
 				prefix = "";
 			}
 			suffix = ".com/";
+		} else if (wikiName.endsWith(".de")) {
```

Comment:
Do we have any other wikiName that ends with a '.' plus alphabets?
If so, this should be changed to take care of that instead of a particular match for '.de'
e.g. use regular expression / pattern matching